Makefile

```diff
@@ -1,10 +1,15 @@
 .PHONY: build
 
 build:
-	./scripts/compile.sh
+	goreleaser build --rm-dist --snapshot
+build-local:
+	goreleaser build --single-target --skip-validate --rm-dist --snapshot
```

Comment:
tu este vyhodim --skip-validate pretoze u snapshotu sa to neuplatnuje, tj moze zavolat 
`goreleaser build --single-target --rm-dist --snapshot`

Reply:
ok :+1: